src/containers/Simularium/index.tsx

```diff
@@ -204,10 +213,14 @@ class App extends React.Component<AppProps, AppState> {
             setViewerStatus,
             clearSimulariumFile,
             setError,
+            conversionStatus,
         } = this.props;
         return (
             <Layout className={styles.container}>
                 <div ref={this.interactiveContent}>
+                    {conversionStatus !== CONVERSION_INACTIVE && (
+                        <ConversionForm />
+                    )}
```

Comment:
Now that the conversion process doesn't have its own path, we are rendering the conversion form here in `Simularium` because the viewer itself doesn't really need to know anything about conversion state, it just needs to handle the converted trajectory. Styling is also much easier here above the `ViewerPanel` layout.